MapDestroyTest.destroyAllReplicasIncludingBackups

```
java.lang.AssertionError: null
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at com.hazelcast.map.MapDestroyTest.assertAllPartitionContainersAreEmpty(MapDestroyTest.java:86)
	at com.hazelcast.map.MapDestroyTest.access$000(MapDestroyTest.java:45)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-IbmJDK1.6/com.hazelcast$hazelcast/782/testReport/junit/com.hazelcast.map/MapDestroyTest/destroyAllReplicasIncludingBackups/

--- a/hazelcast/src/test/java/com/hazelcast/map/MapDestroyTest.java
+++ b/hazelcast/src/test/java/com/hazelcast/map/MapDestroyTest.java
@@ -52,6 +52,13 @@ public class MapDestroyTest extends HazelcastTestSupport {
     public void setUp() {
         TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);
         Config config = getConfig();
+
+        // Set Max Parallel Replications to max value, so that the initial partitions can sync as soon possible.
+        // Due to a race condition in object destruction, it can happen that the sync operation takes place
+        // while a map is being destroyed which can result in a memory leak. This will be overhauled as part of
+        // object lifecycle revamp.
+        // For more details, see: https://github.com/hazelcast/hazelcast/issues/7132
+        config.setProperty(GroupProperty.PARTITION_MAX_PARALLEL_REPLICATIONS, String.valueOf(Integer.MAX_VALUE));
         instance1 = factory.newHazelcastInstance(config);
         instance2 = factory.newHazelcastInstance(config);
     }

